syntax = "proto3";

message userRequest
{
    string username = 1;
    string password = 2;
}

message userResponse 
{
    int32 status = 1;
    string message = 2;
}

service UserService
{
    rpc AddUser(userRequest) returns (userResponse);
    rpc DeleteUser(userRequest) returns (userResponse);
}



